hydra.yaml

```
@@ -0,0 +1,63 @@
+package:
+  name: hydra
+  version: 2.3.0
+  epoch: 0
+  description: OpenID Certified OAuth 2.0 Server and OpenID Connect Provider
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "1" # Needed for SQLite support
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ory/hydra
+      tag: v${{package.version}}
+      expected-commit: ee8c339ddada3a42529c0416897abc32bad03bbb
+
+  # - uses: go/install
+  #   with:
+  #     package: .
+  #     ldflags: -s -w -X 'github.com/ory/hydra/cmd.Version=${{package.version}}' -X 'github.com/ory/hydra/cmd.Commit=${EXPECTED_COMMIT}' -X 'github.com/ory/hydra/cmd.Date=$(date -u +%Y-%m-%dT%H:%M:%SZ)'
+  #     tags: sqlite,json1,hsm
+  - runs: |
+      go install -tags sqlite,json1,hsm \
+        -ldflags "-s -w \
+        -X github.com/ory/hydra/cmd.Version=${PACKAGE_VERSION} \
+        -X github.com/ory/hydra/cmd.Commit=${EXPECTED_COMMIT} \
```

Are these env variables defined somewhere? Example: `${EXPECTED_COMMIT}` and `${PACKAGE_VERSION}`? I think you probably will need this:
 
```suggestion
        -X github.com/ory/hydra/cmd.Version=v${{package.version}} \
        -X github.com/ory/hydra/cmd.Commit=$(git rev-parse HEAD) \
```

---

I just realised these flags dont end up doing what I intended for them to do, that is set the version information. Saw same behavious in upstream build so eliminating flags altogether.

---

Or probably you were looking for this one? https://github.com/ory/hydra/blob/96d4ffe2cf3c6fce40d968f343088be39f8de165/Makefile#L184
My other question is what was the error that we see if using go pipeline, I suppose the ideal way is to use that and if we can't we should comment on why

---

THANK YOU FOR POINTING ME TO THAT
the binary doesnt quite get placed/built using the pipeline, let me try once again and add a comment.